### ISSUE TITLE
Spring.scad - Feature Request: Flat ends

### ISSUE THREAD

**Issue by the-robo** [KitWallace/openscad#4]
Spring.scad
Please add a feature to allow for the ends of the spring to rest on a flat surface.  

Although not my use case, the best example I can think of is the springs under the bed of a 3D printer (or perhaps the replacement springs.)

As it stands (no pun intended) the spring will fall over if stood on one end.

**Comment by KitWallace** [comment]
I agree that would be better - I'll think about it and how to taper the ends of the flat section of the coil
